scripts/codegen/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+FOLDER="./tmp/service-contracts"
+# rm -rf ./tmp
+mkdir -p ./tmp
+
+git -C ${FOLDER} pull || git clone git@github.com:cognitedata/service-contracts.git ${FOLDER}
```

Comment:
Since this is a public repo I'd advise to go against the public endpoints from https://docs.cognite.com/